tsconfig.json

```diff
@@ -1,20 +1,19 @@
 {
   "compilerOptions": {
     "target": "esnext",
-    "module": "commonjs",
-    "allowJs": true,
-    "checkJs": false,
-    "jsx": "preserve",
-    "declaration": true,
-    "outDir": "./types",
-    "emitDeclarationOnly": true,
-    "strict": true,
+    "module": "esnext",
```

Comment:
INFO: Several of these options are override in `yarn run dist:transpile:*`. I updated them here to align with our other TypeScript projects, though I’m open to revisions, since TypeScript libraries like this may have different needs vs TypeScript app code.